proxy-example.py

```diff
@@ -0,0 +1,37 @@
+import stripe
+
+
+stripe.api_key = 'tGN0bIwXnHdwOa85VABjPdSn8nWY7G7I'
+
+print( "Attempting charge..." )
+
+stripe.proxies = {
+    "http": "http://<user>:<pass>@<proxy>:<port>",
+    "https": "http://<user>:<pass>@<proxy>:<port>" }
+
+clients = ( 
+    stripe.http_client.RequestsClient( 
+        verify_ssl_certs= stripe.verify_ssl_certs,
+        proxies= stripe.proxies )
+    ,
+    stripe.http_client.PycurlClient( 
+        verify_ssl_certs= stripe.verify_ssl_certs,
+        proxies= stripe.proxies )
+    ,
+    stripe.http_client.Urllib2Client( 
+        verify_ssl_certs= stripe.verify_ssl_certs,
+        proxies= stripe.proxies ) )
+
+for c in clients:
+    stripe.default_http_client = c
+    resp = stripe.Charge.create(
+        amount=200,
+        currency='usd',
+        card={
+            'number': '4242424242424242',
+            'exp_month': 10,
+            'exp_year': 2019
+        },
+        description='customer@gmail.com'
+    )
+    print( 'Success: %s, %r' % ( c.name, resp, ) )
```

Comment:
One thing that might be a good idea is to move this file and `example.py` into their own `examples` directory so we don't have a bunch of extras hanging around at root, but I can handle that after this is merged.